library/spdm_requester_lib/libspdm_req_send_receive.c

```diff
@@ -668,7 +668,8 @@ libspdm_return_t libspdm_receive_spdm_response(libspdm_context_t *spdm_context,
             spdm_response->request_response_code != SPDM_FINISH_RSP &&
             spdm_response->request_response_code != SPDM_PSK_EXCHANGE_RSP &&
             spdm_response->request_response_code != SPDM_ENCAPSULATED_REQUEST &&
-            spdm_response->request_response_code != SPDM_ENCAPSULATED_RESPONSE_ACK
+            spdm_response->request_response_code != SPDM_ENCAPSULATED_RESPONSE_ACK &&
+            spdm_response->request_response_code != SPDM_CSR
```

Comment:
Instead of check what is allowed, can we check what is not allowed?

For example, if request_response_code is GET_VERSION, GET_CAPABILITYIES, then return error.

That can avoid changing for new command, which will most likely support chunking